doc/nrf/releases/release-notes-2.4.0.rst

```diff
@@ -0,0 +1,955 @@
+    * Hardware `Errata 255`_ of the nRF52833 chip.
+    * Hardware `Errata 256`_ of the nRF52820 chip.
+    * Hardware `Errata 257`_ of the nRF52811 chip.
+    * Hardware `Errata 117`_ of the nRF5340 chip.
+
+Drivers
+=======
+
+This section provides detailed lists of changes by :ref:`driver <drivers>`.
+
+* Added :ref:`nrf700x_wifi`.
+
+Libraries
+=========
+
+This section provides detailed lists of changes by :ref:`library <libraries>`.
+
+Binary libraries
+----------------
+
+* Added the standalone :ref:`lib_bt_ll_acs_nrf53_readme` library, originally a part of the :ref:`nrf53_audio_app` application.
+* :ref:`liblwm2m_carrier_readme` library:
+
+  * Updated to v3.2.0.
+    See the :ref:`liblwm2m_carrier_changelog` for detailed information.
+
+Bluetooth libraries and services
+--------------------------------
+
+* :ref:`bt_le_adv_prov_readme` library:
+
+  * Added API to enable or disable the Swift Pair provider (:c:func:`bt_le_adv_prov_swift_pair_enable`).
+
+* :ref:`bt_fast_pair_readme` library:
+
+  * Added:
+
+    * The :c:func:`bt_fast_pair_info_cb_register` function and the :c:struct:`bt_fast_pair_info_cb` structure to register Fast Pair information callbacks.
+      The :c:member:`bt_fast_pair_info_cb.account_key_written` callback can be used to notify the application about the Account Key writes.
+    * The :kconfig:option:`CONFIG_BT_FAST_PAIR_STORAGE_USER_RESET_ACTION` Kconfig option to enable a custom user reset action that executes together with the Fast Pair factory reset operation triggered by the :c:func:`bt_fast_pair_factory_reset` function.
+
+  * Updated:
+
+    * The salt size in the Fast Pair not discoverable advertising from 1 byte to 2 bytes, to align with the Fast Pair specification update.
+    * The :kconfig:option:`CONFIG_BT_FAST_PAIR_CRYPTO_OBERON` Kconfig option is now the default Fast Pair cryptographic backend.
+
+* :ref:`nrf_bt_scan_readme` library:
+
+  * Fixed the output arguments of the :c:func:`bt_scan_filter_status_get` function.
+    The :c:member:`bt_filter_status.manufacturer_data.enabled` field is now correctly set to reflect the status of the filter when the function is called.
+
+Debug libraries
+---------------
+
+* Added the :ref:`etb_trace` library for instruction traces.
+
+Modem libraries
+---------------
+
+* :ref:`lte_lc_readme` library:
+
+  * Added the Kconfig option :kconfig:option:`CONFIG_LTE_PSM_REQ` that automatically requests PSM on modem initialization.
+    If this option is disabled, PSM will not be requested when attaching to the LTE network.
+    This means that the modem's NVS (Non-Volatile Storage) storage contents are ignored.
+
+  * Updated:
+
+    * The Kconfig option :kconfig:option:`CONFIG_LTE_EDRX_REQ`.
+      The option will now prevent the modem from requesting eDRX in case the option is disabled, in contrast to the previous behavior, where eDRX was requested even if the option was disabled (in the case where the modem has preserved requesting eDRX in its NVS storage).
```

Comment:
```suggestion
      The option will now prevent the modem from requesting eDRX in case the option is disabled, in contrast to the previous behavior, where eDRX was requested even if the option was disabled (in the case where the modem has preserved requesting eDRX in its non-volatile storage).
```